.env.dev

```
@@ -0,0 +1,17 @@
+NEO4J_USER=neo4j
+NEO4J_PASSWORD=test-password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEO4J_PASSWORD key should go before the NEO4J_USER key

(UnorderedKey)